dnf-behave-tests/features/plugins-core/config-manager.feature

```diff
@@ -11,6 +11,20 @@ Background:
         | enabled     | 0        |
 
 
+Scenario: when run without arguments
```

Comment:
We track bugs related to tests with tags, e.g. ``@bz1782822``. Can you please add the tag to this scenario?